### PR TITLE
Fix computed expr

### PR DESCRIPTION
**Pull request by bomsy**
Associated Issue: #5378 

### Summary of Changes

* Use the value instead of the name  if the Property of a computed member expression is a literal

### CODE REVIEW

**Comment by jasonLaster** [comment]
awesome!

**Comment by jasonLaster** [comment]
@bomsy what do you want to do about the snapshot?

**Comment by bomsy** [comment]
i'll confirm and update the snapshot!

File: src/workers/parser/tests/__snapshots__/getSymbols.spec.js.snap [hunk 1]
```diff
@@ -380,7 +380,7 @@ callExpressions:
 memberExpressions:
 [(7, 35), (7, 42)] [(7, 31), (7, 42)] arr.entries entries
 [(8, 10), (8, 13)] [(8, 2), (8, 13)] console.log log
-[(16, 34), (16, 46)] [(16, 19), (16, 47)] prefsBlueprint.accessorName accessorName
+[(16, 34), (16, 46)] [(16, 19), (16, 47)] prefsBlueprint.[accessorName] accessorName
```
Comment (jasonLaster) [inline src/workers/parser/tests/__snapshots__/getSymbols.spec.js.snap]: hmm, does this work?  `prefsBlueprint.[accessorName]`

File: src/workers/parser/tests/__snapshots__/getSymbols.spec.js.snap [hunk 2]
```diff
@@ -503,51 +503,65 @@ variables:
...
-[(23, 29), (23, 43)] [(23, 13), (23, 43)]  secondProperty
-[(23, 27), (23, 28)] [(23, 13), (23, 28)]  c
-[(23, 18), (23, 24)] [(23, 13), (23, 24)] obj2.doEvil doEvil
+[(7, 40), (7, 46)] [(7, 20), (7, 46)] collection.books.[1].author author
+[(7, 37), (7, 38)] [(7, 20), (7, 39)] collection.books.[1]
+[(7, 31), (7, 36)] [(7, 20), (7, 36)] collection.books books
+[(8, 42), (8, 48)] [(8, 21), (8, 48)] collection.books.[ab].editor editor
+[(8, 38), (8, 40)] [(8, 21), (8, 41)] collection.books.[ab] ab
+[(8, 32), (8, 37)] [(8, 21), (8, 37)] collection.books books
+[(9, 66), (9, 74)] [(9, 28), (9, 74)] collection.genres.[\\"sci-fi\\"].movies.[0].director director
```
Comment (jasonLaster) [inline src/workers/parser/tests/__snapshots__/getSymbols.spec.js.snap]: same here: `collection.genres.[\\"sci-fi\\"].movies.[0].director` director

File: src/workers/parser/tests/__snapshots__/getSymbols.spec.js.snap (hunk 1 above)
Comment (bomsy) [reply src/workers/parser/tests/__snapshots__/getSymbols.spec.js.snap]: ummm nope ... it shouldn't ,hmmmm ... how did i missed this. :) will look into it. thanks!

**Comment by bomsy** [comment]
Closing this! will open another PR, with an updated rethought solution.

**Pull request closed without merging**